srcpkgs/wayfire/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wayfire'
+pkgname=wayfire
+version=0.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake"
+makedepends="wlroots-devel glm cairo-devel wf-config libcap-devel"
+short_desc="3D wayland compositor"
+maintainer="Young Jin Park <youngjinpark20@gmail.com>"
+license="MIT"
+homepage="https://wayfire.org"
+distfiles="https://github.com/WayfireWM/wayfire/archive/v${version}.tar.gz"
+checksum=ce6ed0ba62296992f4a0c05b92d05b33911d1d192346611df845414ffd9a18b8
+
+post_install() {
+	vcopy wayfire.ini.default /usr/share/wayfire
```

Comment:
Can it not read /etc/wayfire/config like sway does?

Reply:
yeah it's honestly not too great in upstream

Reply:
Oh it is okay, just wanted to make sure. Also, I think using vinstall might be more robust than vcopy here.

Reply:
you're really killing me here.... jk :smile: thanks for your feedback, fixed